scripts/zones/Jugner_Forest/IDs.lua

```diff
@@ -66,8 +66,13 @@ zones[xi.zone.JUGNER_FOREST] =
             [17203447] = 17203448,
         },
 
-        KING_ARTHRO = 17203216,
-        FRAELISSA   = 17203447,
+        KING_ARTHRO          = 17203216,
+        FRAELISSA            = 17203447,
+        DUESSA_START         = 17203449,
```

Comment:
similar to meteomaulers adds, you only really need to know the offset from Fradubio's ID